docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,55 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+Here is a summary of the guide. See each section for additional details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) following the instructions for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Start the environment:
+
+ ```
+ wp-env start
```

Comment:
Should we clarify that this command would need to be run by `cd`ing to a different directory if they wish to use it for an existing plugin/codebase? (e.g. for developing gutenberg?)

The instance will be different for each time they run it, so any changes they make to this instance would not apply if they started a different one from the Gutenberg directory.

Reply:
if the user just does this first without realizing any of the later information, they might get conflicting port numbers, which is why I mention it.

Reply:
I can try to clarify, we can also add a troubleshooting section, list a couple of common errors